bokeh/util/notebook.py

```diff
@@ -1,14 +1,16 @@
-''' Functions useful for loading Bokeh code and data in IPython notebooks.
+''' Functions useful for loading Bokeh code and data in Jupyter/Zeppelin notebooks.
 
 '''
 from __future__ import absolute_import
 
 _notebook_loaded = None
 
-def load_notebook(resources=None, verbose=False, hide_banner=False, load_timeout=5000):
+def load_notebook(notebook_type='jupyter', resources=None, verbose=False, hide_banner=False, load_timeout=5000):
```

Comment:
should make this new parameter at the end instead of the beginning (better for backward compat)